src/sentry/api/endpoints/project_rule_task_details.py

```diff
@@ -35,7 +36,7 @@ def get(self, request: Request, project, task_uuid) -> Response:
                 rule = Rule.objects.get(
                     project=project,
                     id=int(rule_id),
-                    status__in=[RuleStatus.ACTIVE, RuleStatus.INACTIVE],
+                    status=ObjectStatus.ACTIVE,
```

Comment:
@ceorourke so before we would query inactive rules?

Reply:
Yeah I'm not really sure why, might have been an intention long ago to have an active/inactive rule feature (which is now mute alerts) but we only have 1 inactive rule and it's going to be deleted in the migration I linked.